src/Core/Managed/Shared/Extensibility/Implementation/IClock.cs

```diff
@@ -4,6 +4,6 @@
 
     internal interface IClock
     {
-        DateTimeOffset Time { get; }
+        TimeSpan Time { get; }
```

Comment:
This Interface is not used elsewhere.
Any advantage of using Ticks instead of storing TimeSpan? StopWatch can return Ticks or TimeSpan - i used Timespan as Duration is already in TimeSpan.
Not clear about where the two conversions are happening?